### ISSUE TITLE
 Deep autoencoder neural networks for gene ontology annotation predictions

### ISSUE THREAD

**Issue by agitter** [greenelab/deep-review#66]
http://doi.org/10.1145/2649387.2649442

**Comment by cgreene** [comment]
The authors converted the gene ontology into a binary matrix. Each row is a gene, and each column is a GO term. Features are set to 1 if that gene is annotated to that GO term.

The authors then train an autoencoder on this. They use a hold out from pre-2009 annotations to fit hyperparameters. Then they use 2009-2013 annotations as a full test set.

Basically the framing of this is saying: if a gene has these GO annotations, are there other annotations that it is likely to have? They also compare to truncated SVD.

The novel & interesting predictions included at the end (Table 4) are to relatively generic terms. The topic also seems a little bit peripheral to this review. I'm going to close this now, but if this starts to come up as we write the paper we should reconsider.
